js/product-drop.js

```diff
@@ -29,6 +29,18 @@ export const ProductDropAPI = superclass =>
             const productDrop = await this.delete(url);
             return productDrop;
         }
+
+        async like(id, payload) {

```

Comment:
Because this is used as a mixin it should probably have a suffix to avoid future collisions
```suggestion
        async likeProductDrop(id, payload) {
```